### PR TITLE
Use cockpit library components and webpack helpers from cockpit.git

### PR DESCRIPTION
**Pull request by martinpitt**
Drop the cockpit-po-plugin.js, sassc-loader.js, variables.scss etc.
code copies, and check them out from a stable cockpit tag, similarly to
how we already get the PatternFly CSS.

Also move them from lib/ to src/lib/ for consistency with starter-kit.

Install bootstrap-sass npm module, as the current variables.scss imports
it. It's not really necessary for cockpit-ostree, but scss compilation
would fail otherwise. (Let's try to drop it from cockpit's version.)

Ported from starter-kit commit bb064229967.

### CODE REVIEW

**Comment by martinpitt** [comment]
I tested this visually locally, and it looks fine to me.

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "@babel/preset-react": "^7.0.0",
     "babel-eslint": "^10.0.3",
     "babel-loader": "^8.0.6",
+    "bootstrap-sass": "^3.4.1",
```
Comment (KKoukiou) [inline package.json:22]: I really wonder how it worked at the podman repo!??

**Review by KKoukiou — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (KKoukiou) [reply package.json:22]: Oh I know, we don't include there the form-layout.scss that imports the variables.scss file.

**Comment by KKoukiou** [comment]
Btw usage of form-layout layout should be replaced with Form component from PF4. This file is from a pre PF4 era.

**Pull request merged**